cmd/snap-bootstrap/main.go

```diff
@@ -22,33 +22,45 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/snapcore/snapd/cmd/snap-bootstrap/bootstrap"
+	"github.com/jessevdk/go-flags"
+
 	"github.com/snapcore/snapd/osutil"
 )
 
+var (
+	shortHelp = "Bootstrap a Ubuntu Core system"
+	longHelp  = `
+snap-bootstrap is a tool to bootstrap a Ubuntu Core
```

Comment:
```
snap-bootstrap is a tool for bootstrapping Ubuntu Core
from an ephemeral system such as initramfs.
```
Though I'd trust @degville with coming up with the right wording more than myself.

Reply:
sounds good! - my slight tweak:
`snap-bootstrap is a tool to bootstrap Ubuntu Core from ephemeral systems such as initramfs.`

Reply:
Thank you so much guys!